app/Repositories/BooksRepository.ts

```diff
@@ -14,25 +14,26 @@ export default class BooksRepository {
       autores: authors,
     }
     books.push(book)
-    return books
+    return books[books.length - 1]
```

Comment:
Pq não retornar apenas `book` direto?
```suggestion
    return book
```